Fix the following compile error, observed with clang-18 on jammy and noble,
by adding a missing template argument:

  FAILED: obj/pdf/internal/pdfium_api_wrappers.o
  [...]
  ../../pdf/pdfium/pdfium_api_wrappers.cc:150:32: error: alias template 'PDFiumAPIStringBufferAdapter' requires template arguments; argument deduction only allowed for class templates
    150 |   PDFiumAPIStringBufferAdapter adapter(&name, expected_size,
        |                                ^
  ../../pdf/pdfium/pdfium_api_string_buffer_adapter.h:173:1: note: template is declared here
    173 | using PDFiumAPIStringBufferAdapter =
        | ^

--- a/pdf/pdfium/pdfium_api_wrappers.cc
+++ b/pdf/pdfium/pdfium_api_wrappers.cc
@@ -147,7 +147,7 @@ std::u16string GetPageObjectMarkName(FPD
 
   // Number of characters, including the NUL.
   const size_t expected_size = base::checked_cast<size_t>(buflen_bytes / 2);
-  PDFiumAPIStringBufferAdapter adapter(&name, expected_size,
+  PDFiumAPIStringBufferAdapter<std::u16string> adapter(&name, expected_size,
                                        /*check_expected_size=*/true);
   unsigned long actual_buflen_bytes = 0;  // NOLINT(runtime/int)
   bool result =
